pkg/neorpc/events.go

```diff
@@ -86,3 +106,62 @@ func (e *EventID) UnmarshalJSON(b []byte) error {
 	*e = id
 	return nil
 }
+
+// Matches filters our given EventContainer against EventComparator filter.
+func Matches(f EventComparator, r EventContainer) bool {
```

Comment:
This doesn't belong to `neorpc`. Maybe `neorpc/result`. Or even a package of its own.